test/models/user_test.rb

```diff
@@ -138,6 +138,28 @@ class UserTest < ActiveSupport::TestCase
     assert_not user.depressed?
   end
 
+  test '.depressed_reports' do
+    no_depressed_user = users(:kimura)
+    first_depressed_user = users(:hatsuno)
+    second_depressed_user = users(:kensyu)
+
+    ids = [
+      no_depressed_user.id,
+      first_depressed_user.id,
+      second_depressed_user.id
+    ]
+    users = User.where(id: ids)
```

Comment:
```suggestion
    users = [no_depressed_user, first_depressed_user, second_depressed_user]
```

`User.where(id: ids)`でusersを取得する必要がある理由があれば知りたいです！
なければ配列に入れてしまった方がコードがスッキリすると思いました〜

Reply:
@yana-gi こちらは `User.depressed_reprots` のなかで `ids`メソッドをつかっているので、`ActiveRecord::Relation` として渡したいため、`users`を取得していますー :+1:

Reply:
なるほど……！勉強になりました〜ありがとうございます😄